tests/unit/inference/test_inference.py

```diff
@@ -261,6 +261,11 @@ def validate_test(model_w_task, dtype, enable_cuda_graph, enable_triton):
 
 
 @pytest.mark.inference
+@pytest.mark.parametrize("model_w_task", [
```

Comment:
Can we make sure to remove this parametrize since it'll override the `model_w_task` fixture, which is the more comprehensive enumeration of tests?

Reply:
yes I removed it as before